src/index.html

```diff
@@ -16,17 +29,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="search-panel">
+      <form
+        class="search-form big"
+        data-qa="keypress"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+      <form
+        class="search-form small"
+        data-qa="small"
+      >
+        <input
+          class="search__input search__input--small"
+          type="text"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute is set to `hover` for the small search form, which is incorrect according to the task requirements. It should be `small`.